src/components/Repos.js

```diff
@@ -129,27 +162,58 @@ const Repos = ({ materiaSelected, repos, materias, partialLoading }) => {
   );
 };
 
-const SortFeature = ({ sortOption, setSortOption }) => {
+const DisplayFeature = ({ sortOption, setSortOption, languageFilter, setLanguageFilter, allLanguages }) => {
   return (
-    <Tooltip label={`Ordenado por ${sortOption.longName}`}>
-      <IconButton
-        position="absolute"
-        top="0"
-        right="0"
-        variant="link"
+    <Box
+      position="sticky"
+      top={0}
+      left={0}
+      right={0}
+      display="flex"
+      alignItems="center"
+      justifyContent="space-between"
+      p={4}
+      pt={4}
```

Comment:
```suggestion
      p={4}
```

quedo redundante el pt=4